product/views.py

```diff
@@ -1,3 +1,63 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+from django.db.models import Prefetch
+from user.models import Creator,User
+from request.models import CourseRequest
+from product.models import (
+    Course,
+    Category,
+    SubCategory,
+    ProductImage,
+    Product,
+    ProductLike
+)
+
+#클래스 좋아요 기능
+class CourseLikeView(View):
+   # @id_auth
```

Comment:
나중에 해야될 일은 주석이 아니라 `TODO`를 사용해주세요.
```suggestion
   # TODO : Add @id_auth
```